WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -16045,14 +16069,6 @@ e_armenia = {
 		zoroastrian_group = 200 # Crusade target weight
 		
 		# Creation/usurpation trigger
-		allow = {
-			hidden_tooltip = {
-				OR = {
-					ai = no
-					religion_group = christian
-				}
-			}
-		}
```

Comment:
In my code, there's nothing at the sixteen thousandth fifty fifth line to have been removed:

`				b_tercan = {
				}
				b_kirklarkilisesi = {
				}
			}
		}
	}
	k_caucasian_albania = {
		color={ 172 82 55 }
		color2={ 255 255 255 }
		
		capital = 668 # Shirvan
		
		culture = armenian`